Tests/UIWebViewHookTests.m

```diff
@@ -148,17 +140,6 @@ - (void)testEndMetricCalledOnWebViewDidFailLoadWithError
 
 // MARK: Delegate inheritance tests
 
-- (void)testWebViewDelegateSubclassedMethodCallsSuper
-{
-    id mockTracker = OCMPartialMock(self.trackingManager.tracker);
-    
-    WebViewControllerChild *child = [self childWebViewController];
-    [child.webViewInChild.delegate webViewDidStartLoad:child.webViewInChild];
-    
-    OCMVerify([mockTracker prolongMetric]);
-    [mockTracker stopMocking];
-}
-
```

Comment:
Instead of dropping this test can we replace it with a webViewDidFinishLoad check?

Reply:
The unit test with a webViewDidFinishLoad has already existed.